espflash/src/error.rs

```diff
@@ -283,6 +287,25 @@ impl From<SlipError> for Error {
     }
 }
 
+impl From<TryFromSliceError> for Error {
+    fn from(err: TryFromSliceError) -> Self {
+        Self::InvalidResponse(err.to_string())
```

Comment:
I don't understand why this doesn't convert into `Error::TryFromSlice`, xor why that variant hasn't been removed.

Reply:
Without having it, [this](https://github.com/esp-rs/espflash/blob/main/espflash/src/flasher/mod.rs#L110) (and others) is not happy.

Reply:
But does this have to convert into the wrong error variant?

Reply:
Oh, I see now - you are right, probably copy-paste error.